doc/classes/SplitContainer.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="SplitContainer" inherits="Container" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A container that splits two child controls horizontally or vertically and provides a grabber for adjusting the split ratio.
+		A container that arranges child controls horizontally or vertically and provides grabbers for adjusting the split ratios between them.
 	</brief_description>
 	<description>
-		A container that accepts only two child controls, then arranges them horizontally or vertically and creates a divisor between them. The divisor can be dragged around to change the size relation between the child controls.
+		A container that arranges child controls horizontally or vertically and creates divisors between them. The divisors can be dragged around to change the size relations between the child controls.
```

Comment:
Why did you repeat the above, but with divisors instead of grabbers? 🙃

Reply:
That's just how it was beforehand. Changed to use grabbers.